dali/python/nvidia/dali/plugin/mxnet.py

```diff
@@ -99,6 +97,13 @@ def __init__(self,
         self._data_batches = [[None] for i in range(self._num_gpus)]
         self._counter = 0
         self._current_data_batch = 0
+        self._output_names_map = [x[0] for x in output_map]
+        self._output_categories_map = [x[1] for x in output_map]
+        self._output_categories = {DALIGenericIterator.DATA_TAG, DALIGenericIterator.LABEL_TAG}
+        assert set(self._output_categories_map) <= self._output_categories, \
```

Comment:
Shouldn't it be also:
```suggestion
        assert len(set(self._output_categories_map)) <= len(self._output_categories), \
```

Reply:
Nope, because I want to change if there are only allowed elements in `_output_categories_map`, and not if the number of the categories is ok.

Reply:
Ok